src/planwise/model/projects2.clj

```diff
@@ -12,11 +12,13 @@
   [config]
   (-> config
       (default [:demographics :target] nil)
+      (default [:analysis-type] "budget")
       (default [:actions :budget] nil)
       (default [:coverage :filter-options] {})
       (default [:providers :capacity] 1)))
 
 (s/def ::target (s/nilable number?))
+(s/def ::analysis-type (s/nilable string?))
```

Comment:
Is there a reason to make analysis-type optional?

Existing project settings will be updated on get-project/2 thanks to the apply-default.

Reply:
No, there's no reason. I'll update.